src/util/psbt/mod.rs

```diff
@@ -750,7 +750,7 @@ mod tests {
                                 vout: 1,
                             },
                             script_sig: hex_script!("160014fe3e9ef1a745e974d902c4355943abcb34bd5353"),
-                            sequence: 4294967295,
+                            sequence: Sequence::from_consensus(4294967295),
```

Comment:
This (and some others) is actually equal to `Sequence::MAX`

Reply:
Geek points for @Kixunil for noticing 2^32-1 = 4294967295, I missed that :)